src/services/dispatchers.ts

```diff
@@ -107,22 +102,42 @@ async function extractDispatchersFromFile(path: string): Promise<DispatcherCreat
   }
 }
 
-function assertRequiredFields(dispatcher: DispatcherCreation) {
-  // Ensure all dispatchers have names
-  if (!dispatcher.name) {
-    throw new Error('Invalid Dispatcher: Dispatcher without a name 𝖷');
-  }
+function validateDispatchers(dispatchers: DispatcherCreation[]) {
+  let hasErrors = false;
 
-  const hasActions = Array.isArray(dispatcher.actions) && dispatcher.actions.length > 0;
-  if (!hasActions) {
-    throw new Error(`Invalid Dispatcher: ${dispatcher.name} needs at least one action 𝖷`);
-  }
+  console.group(blue('Validating Dispatchers:'));
+  dispatchers.forEach((dispatcher, index) => {
+    const displayName = dispatcher?.name ? `[${index}]: ${dispatcher.name}` : `[${index}]: NO_NAME`;
+    const errors = [];
 
-  // Ensure all actions have names
-  const hasValidActions = dispatcher.actions.every(action => action.name);
-  if (!hasValidActions) {
-    throw new Error(`Invalid Dispatcher: ${dispatcher.name} has actions without a name 𝖷`);
-  }
+    // Ensure all dispatchers have names
+    if (!dispatcher.name) {
+      errors.push('No name');
+    }
 
-  console.log(green(`Validated Dispatcher: ${dispatcher.name} ✓`));
+    const hasActions = Array.isArray(dispatcher.actions) && dispatcher.actions.length > 0;
+    if (!hasActions) {
+      errors.push('Needs at least one action');
```

Comment:
This error isn't descriptive enough compared to the check that we do? We also check if it's an array? 🤔 

Could be misleading?